cscs-checks/tools/profiling_and_debugging/ddt.py

```diff
@@ -40,6 +40,7 @@ def __init__(self, lang, extension, **kwargs):
         }
         self.maintainers = ['MK', 'JG']
         self.tags = {'production'}
+        self.post_run = ['ddt -V ; which ddt ;']
```

Comment:
Is this line needed just for checking or can we put another sanity check for this?